test/functional/test_framework/wallet.py

```diff
@@ -182,6 +183,26 @@ def sendrawtransaction(self, *, from_node, tx_hex):
         from_node.sendrawtransaction(tx_hex)
         self.scan_tx(from_node.decoderawtransaction(tx_hex))
 
+    def send_large_transactions(
+        self, node, array_of_large_tx, no_of_tx_ids, fee_rate
+    ):
+        # Create large transactions by appending txouts in vout
+        no_of_tx_created = 0
```

Comment:
I'm not really seeing the value of this local variable. if there was a way to return early in the for loop, incrementing the number could be a way to keep track of successes. but since there isn't any logic of the sort, any unsuccessful attempts would simply cause an error (eg. by `sendrawtransaction` throwing an exception). so `assert_equal(no_of_tx_created,no_of_tx_ids)` isn't actually increasing any guarantees from this code. 

my suggestion would be to remove the `no_of_tx_created` logic from this function. you could return `no_of_tx_ids` if you want to maintain the current return value, or just update the call site since that is where the number of transactions to create gets passed in anyways.

Reply:
Done! Thanks